app/views/your_contact_details/your_contact_details.scala.html

```diff
@@ -93,6 +96,8 @@
                 text=messages("cf.contact-details.report-change-link")
             )
         )
+
+        @messages("cf.contact-details.report-change-post-link-text")
```

Comment:
Once the newTabLink is updated to use postLinkMessage, this message  key will be passed as parameter